### PR TITLE
chore(docs): update the version matrix

### PR DESCRIPTION
**Pull request by ch0ppy35**
Please provide a description for what this PR is for.

- Note that v1.11 is no longer supported
- Note the date when v1.12 is no longer supported

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by craigbox** [comment]
/ok-to-test

Thanks for picking this up!  

/cc @Kmoneal to confirm

**Comment by Kmoneal** [comment]
Yep, looks good to me. Thank you!

**Review by craigbox — APPROVED** [review]

File: content/en/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 | master          | No, development only |                   |                          |                                |                              |
 | 1.14            | Yes                  | May 24, 2022      | ~January 2023 (Expected) | 1.21, 1.22, 1.23, 1.24         | 1.16, 1.17, 1.18, 1.19, 1.20 |
 | 1.13            | Yes                  | February 11, 2022 | ~October 2022 (Expected) | 1.20, 1.21, 1.22, 1.23         | 1.16, 1.17, 1.18, 1.19       |
-| 1.12            | Yes                  | November 18, 2021 | ~June 2022 (Expected)    | 1.19, 1.20, 1.21, 1.22         | 1.16, 1.17, 1.18             |
-| 1.11            | Yes                  | August 12, 2021   | Mar 25, 2022             | 1.18, 1.19, 1.20, 1.21, 1.22   | 1.16, 1.17                   |
+| 1.12            | Yes                  | November 18, 2021 | Jul 12, 2022             | 1.19, 1.20, 1.21, 1.22         | 1.16, 1.17, 1.18             |
```
Comment (ericvn) [inline content/en/docs/releases/supported-releases/index.md:59]: I would have left the Expected in as the date is tentative.

**Pull request merged**